### ISSUE TITLE
jan 11 2024 bug: score total is being added multiple times

### ISSUE THREAD

**Issue by AccelQuasarDragon** [ShootingStarDragon/FastCVApp#429]
- [x] another issue, apparently score was carrying over BETWEEN closing and opening again from terminal, wtf > #430
- [x] The correct question is when do i say already graded = false, aka when do i reset the grading?
- [x] Currently i do it when the future frame is found but it can be  multiple times in the correct interval (because of sequence conditions)
- [x] When to say graded = false?
- [x] Only once in the correct interval (outside of frading period)
- [x] So within the interval, do once, outside of interval, reset....
- [x] =-=-=-=-=-=
- [x] "real" bug:
- [x] already graded is being set to false repeatedly in `if test_time > show_future_pose_time:` block ~L612
- [x] In the if statement check for already graded?
- [x] That's already thw case, but still score is too high (gets added multiple times...)
- [x] Check pid again? 
- [x] Check for loop structure or smth
- [x] =-=-=-=-=-=
- [x] go back to while = True and remove the sleep(0.1) from open_camerapipeline
- [x] CHECK THE EARLIEST TIME GRADED IS TRUE
- [x] goal 1: figure out where total score is being changed
	- [x] do that by going into every step of the process, I believe it's ads but unsure, 
		- [x] turn off everything
			- [x] turn off ALL points at which i add
				- [x] if turning off the 1 point doesn't do it, it gets added somewhere else
					- [x] > CONFIRMED: that is the only point where I add to total, so it is being run multiple times
				- [x] it works right
				- [x] so what's the deal?
				- [x] u know it's one PID, but it gets added multiple times... what's up with that?
					- [x] yes because already graded was False multiple times in the interval that it tries to grade, so I set already graded to True ONCE in the grading interval, then reset outside the grading interval, hoping this works for all cases
				- [x] make an if statement checking for already graded then stuff in the score increase + other stuff to do in it
				- [x] maybe not? maybe it's that the check keeps going through multiple times -> score keeps increasing too much, is that what i want or not?
		- [x] turn on things one by one and check it out
- [x] total score is too high.... > it's high because #421, if I tried the total max score would be a lot higher
	- [x] score is being increased multiple times because of multiprocessing?
	- [x] my mind is melted, idk why....
- [x] Might be related to flicker ans skipping first few iterations problem.... FUCK
	- [x] yeah i think so too, it's smth to do with open_camerapipeline being fucked up
- [x] NOTES:
	- [x] total score was 0 
	- [x] then first change:
		- [x] scam?I 300
		- [x] then later
		- [x] C>D>E>F>G>H
		- [x] then increased to
		- [x] scam?I 600
		- [x] so it's increasing even if nothing is happening, fuck really need to fix this + fix flicker
	- [x] scam?I is 2nd to last screenshot
	- [x] so there is a huge if statement, multiple if statements in a while loop.... like it had seen that shared_camera_subprocess_dictVAR["scoredictVAR"].keys() existed and just straight up added without regard to any condition do u know what i mean? like right before the official adding or smth
- [x] fixed the issue, this was NOT the solution
- [x] pressing play == seeking to 0 basically (so touch up/touch down at slider =0)
